data/helpers.d/utils

```diff
@@ -35,7 +35,7 @@ ynh_exit_properly () {
         ynh_clean_setup	# Call the function to do specific cleaning for the app.
     fi
 
-    ynh_die	# Exit with error status
+    exit 1  # Exit with error status
```

Comment:
:grin: ...

A comment :grin: ?

Honestly, I'm quite sure sooner or later we're going to revert to ynh_die if we don't have a comment to explain it was actually made on purpose.